packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -151,6 +151,78 @@ export interface CronPayload {
   cron_org: string;
 }
 
+export interface BotSecrets {
+  privateKey: string;
+  appId: string;
+  secret: string;
```

Comment:
nit: let's rename this `webhookSecret` for clarity

Reply:
Done